tests/base/functions_13.cc

```diff
@@ -23,30 +23,47 @@
 
 #include "../tests.h"
 
-using namespace dealii;
-
 template <typename Number>
-class MyFunctionTime : dealii::FunctionTime<Number>
+class MyFunctionTime : public dealii::FunctionTime<Number>
```

Comment:
ha, that was pretty pointless before -- deriving `private` from an interface class makes little sense...